sdk/opendp/smartnoise/sql/__init__.py

```diff
@@ -25,7 +25,7 @@
 ]
 
 
-def execute_private_query(reader, schema, budget, query):
+def execute_private_query(reader, schema, epsilon, query):
```

Comment:
I'm just now noticing this doesn't pass in delta, which will result in the default delta from PrivateReader being used.  This may be OK, but could be either too conservative or too relaxed, depending on how many rows are in the source data.  The delta term impacts how many records get dropped in the tau thresholding.